MapboxCoreNavigation/RouteController.swift

```diff
@@ -428,6 +432,14 @@ open class RouteController: NSObject {
             outstandingFeedbackEvents.remove(at: index)
         }
     }
+    
+    /**
+     Set the rating and any comment the user may have about their route. Only used when exiting navigaiton.
+     */
+    @objc public func setEndOfRoute(rating: Int, comment: String?) {
+        endOfRouteStarRating = rating
```

Comment:
We might want to add some basic validation to the rating to conform to the spec:

```
Rating from `0-100` for post ride feedback. `-1 = unrated`
5 star rating system:

- 0 stars (unrated) = `-1`
- 1 star = `0`
- 2 stars = `25`
- 3 stars = `50`
- 4 stars = `75`
- 5 stars = `100`
```

Reply:
Validation is happening here:
https://github.com/mapbox/mapbox-navigation-ios/blob/master/MapboxCoreNavigation/MMEEventsManager.swift#L205

Unsure if we're multiplying yet.

Reply:
Yep, we're multiplying correct.